cmd/syncthing/gui.go

```diff
@@ -1019,8 +1019,13 @@ func (s *apiService) getEvents(w http.ResponseWriter, r *http.Request, eventSub
 	qs := r.URL.Query()
 	sinceStr := qs.Get("since")
 	limitStr := qs.Get("limit")
+	timeoutStr := qs.Get("timeout")
 	since, _ := strconv.Atoi(sinceStr)
 	limit, _ := strconv.Atoi(limitStr)
+	timeoutSec, timeoutErr := strconv.Atoi(timeoutStr)
+	if timeoutErr != nil || timeoutSec < 0 { // A timeout of 0 is valid
```

Comment:
OK, sure. I thought it might be nice to support a timeout of 0 (since I can imagine a caller just wanting to know whether any events are available - the Global Changes stuff in the GUI might want to do this, for instance).